var/spack/repos/builtin/packages/libx264/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libx264(AutotoolsPackage):
+    """x264 is a free software library and application for encoding video
+    streams into the H.264/MPEG-4 AVC compression format."""
+
+    homepage = "https://www.videolan.org/developers/x264.html"
+    url      = "https://download.videolan.org/pub/videolan/x264/snapshots/x264-snapshot-20190811-2245.tar.bz2"
+
+    version('264-snapshot-20190902-2245', sha256='413c0e9b59c1a47f51e3133ba10d4b665456a2687eee0b0a0add58c8684fdfd1')
```

Comment:
I think the correct version here should be `20190902`, or even `2019-09-02`. Can you update this and add a `url_for_version`?

Reply:
@mamelara do you agree with this and if so do you think you'll have a chance to update it any time soon?